pkgs/os-specific/linux/targetisns/install_prefix_path.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f46144d..aeac3e4 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -14,10 +14,10 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror")
+ option(SUPPORT_SYSTEMD "Support service control via systemd" OFF)
+ 
+ add_subdirectory(src)
+-install(FILES target-isns.conf DESTINATION /etc/)
++install(FILES target-isns.conf DESTINATION ${CMAKE_INSTALL_PREFIX}/etc/)
+ install(FILES target-isns.8 DESTINATION ${CMAKE_INSTALL_PREFIX}/share/man/man8/)
+ if (SUPPORT_SYSTEMD)
+-  install(FILES target-isns.service DESTINATION /usr/lib/systemd/system/)
++  install(FILES target-isns.service DESTINATION ${CMAKE_INSTALL_PREFIX}/usr/lib/systemd/system/)
```

Comment:
Correct: https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html

Reply:
OK, now the systemd files get actually installed, and end up in `lib/systemd/system`. 
I am a bit hesitant when comes to upstreaming such nix specific patches. It may take more time energy to explain the patch than just keeping in it nixpkg.

Reply:
`CMAKE_INSTALL_PREFIX` is not Nix specific, it's CMake-specific. It is also mentioned in their own tutorial: https://cmake.org/cmake/help/latest/guide/tutorial/index.html

Reply:
Sure, CMAKE_INSTALL_PREFIX is a cmake feature. However, as it is now the patch breaks a standard FHS distro's installation: the config file would end up in e.g. /usr/etc or /usr/local/etc instead of /etc. I am not too familiar with cmake. What would be the right way to do this.